### PR TITLE
fix deploy shell scripts

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,6 @@ bin/
 # IDE config files
 .dead/
 .vscode/
+.idea/
+hack/deploy/cert-*
+hack/deploy/controlplane/
```
Comment (clyang82) [inline .gitignore:24]: `hack/deploy/controlplane/cert`?

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (zhiweiyin318) [reply .gitignore]: done

**Pull request merged**